### PR TITLE
APP-303: Credentials management, noSDK - reviewed.

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: credentialsdemo/resources/projects.xml [hunk 1]
```diff
@@ -129,7 +129,111 @@ If you want to modify the sample application source code or create your own appl
...
+        <description>Kaa Credentials management demo for noSDK</description>
+        <details>
+            <![CDATA[
+<p>
+	This sample application uses the server-side <b>Credentials management</b> feature based on the <a href="http://kaaproject.github.io/kaa/docs/Glossary/#sdk-type" target="_blank">Kaa Java SDK</a>.
+	The application uses the Kaa REST API to generate public and private user keys, provision the credentials, get their status and revoke them.
+</p>
+
+<p>
+	To generate the RSA keys for the user, the <code>org.kaaproject.kaa.common.endpoint.security.KeyUtil</code> class functionality and standard Java functions are used.
```
Comment (Rib47) [inline credentialsdemo/resources/projects.xml]: remove this: "the <code>org.kaaproject.kaa.common.endpoint.security.KeyUtil</code> class functionality and"

File: credentialsdemo/resources/projects.xml [hunk 2]
```diff
@@ -129,7 +129,111 @@ If you want to modify the sample application source code or create your own appl
...
         <destBinaryFile>JCredentialsDemo/build/jar/JCredentialsDemo.jar</destBinaryFile>
     </project>
+    <project id="credentials_demo_noSDK">
+        <name>Credentials demo noSDK</name>
+        <description>Kaa Credentials management demo for noSDK</description>
+        <details>
+            <![CDATA[
+<p>
+	This sample application uses the server-side <b>Credentials management</b> feature based on the <a href="http://kaaproject.github.io/kaa/docs/Glossary/#sdk-type" target="_blank">Kaa Java SDK</a>.
+	The application uses the Kaa REST API to generate public and private user keys, provision the credentials, get their status and revoke them.
```
Comment (Rib47) [inline credentialsdemo/resources/projects.xml]: Insert:
The application uses the Kaa REST API _(wrapped inside **org.kaaproject.kaa.server.common.admin.AdminClient** class)_ ...

**Review by Rib47 — CHANGES_REQUESTED** [review]

File: credentialsdemo/resources/projects.xml [hunk 3]
```diff
@@ -129,7 +129,111 @@ If you want to modify the sample application source code or create your own appl
...
+
+<ul>
+	<li>
+		<code>adminClient.provisionCredentials</code> method is used to provision the credentials.
+	</li>
+	<li>
+		<code>adminClient.revokeCredentials</code> method is used to revoke the credentials.
+	</li>
+	<li>
+		<code>adminClient.getCredentialsStatus</code> method returns the status of credentials.
```
Comment (Rib47) [inline credentialsdemo/resources/projects.xml]: Replace with:
**<code>AdminClient.getCredentialsStatus()</code>** ...

File: credentialsdemo/resources/projects.xml [hunk 4]
```diff
@@ -129,7 +129,111 @@ If you want to modify the sample application source code or create your own appl
...
+</p>
+
+The sample application also uses the following methods:
+
+<ul>
+	<li>
+		<code>adminClient.provisionCredentials</code> method is used to provision the credentials.
+	</li>
+	<li>
+		<code>adminClient.revokeCredentials</code> method is used to revoke the credentials.
```
Comment (Rib47) [inline credentialsdemo/resources/projects.xml]: Replace with:
**<code>AdminClient.revokeCredentials()</code>** ...

File: credentialsdemo/resources/projects.xml [hunk 5]
```diff
@@ -129,7 +129,111 @@ If you want to modify the sample application source code or create your own appl
...
+
+<p>
+	To generate the RSA keys for the user, the <code>org.kaaproject.kaa.common.endpoint.security.KeyUtil</code> class functionality and standard Java functions are used.
+</p>
+
+The sample application also uses the following methods:
+
+<ul>
+	<li>
+		<code>adminClient.provisionCredentials</code> method is used to provision the credentials.
```
Comment (Rib47) [inline credentialsdemo/resources/projects.xml]: Replace with:
**<code>AdminClient.provisionCredentials()</code>** ...

**Review by Rib47 — CHANGES_REQUESTED** [review]

File: credentialsdemo/resources/projects.xml (hunk 2 above)
Comment (hunch7) [reply credentialsdemo/resources/projects.xml]: Fixed.

File: credentialsdemo/resources/projects.xml (hunk 1 above)
Comment (hunch7) [reply credentialsdemo/resources/projects.xml]: Fixed.

File: credentialsdemo/resources/projects.xml (hunk 5 above)
Comment (hunch7) [reply credentialsdemo/resources/projects.xml]: Fixed.

File: credentialsdemo/resources/projects.xml (hunk 4 above)
Comment (hunch7) [reply credentialsdemo/resources/projects.xml]: Fixed.

File: credentialsdemo/resources/projects.xml (hunk 3 above)
Comment (hunch7) [reply credentialsdemo/resources/projects.xml]: Fixed.

**Review by Slash32 — APPROVED** [review]

**Pull request merged**